presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -139,6 +139,7 @@
     private int partitionStatisticsSampleSize = 100;
     private boolean ignoreCorruptedStatistics;
     private boolean collectColumnStatisticsOnWrite;
+    private boolean negativeStatisticCountEnabled;
```

Comment:
add this HiveClientConfig. default value is false, which means that if we meet -1, we think that the statistics value is corrupted.